npp.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## ------- Set up our list of numbers -------
+S = [25, 7, 13, 31, 42, 17, 21, 10]
+
+## ------- Set up our QUBO dictionary -------
+
+# TODO:  Add code here to define your QUBO dictionary
+
+## ------- Run our QUBO on the QPU -------
+# TODO:  Choose QPU parameters
+chainstrength = 1
+numruns = 10
+
+# Run the QUBO on the solver from your config file
+from dwave.system import DWaveSampler, EmbeddingComposite
+
+sampler = EmbeddingComposite(DWaveSampler())
+sample_set = sampler.sample_qubo(Q, chain_strength=chainstrength, num_reads=numruns)
+
+## ------- Return results to user -------
+for sample in sample_set:
+    S1 = [S[i] for i in sample if sample[i] > 0]
+    S0 = [S[i] for i in sample if sample[i] < 1]
```

Comment:
The if-conditions aren't the most intuitive; it took me several minutes to process. Perhaps the following might be better?
```
S1 = [.. if sample[i] == 1]
S0 = [.. if sample [i] == 0]
```

Also, I believe sample is a dictionary? If so, it might be easier to read if we just iterate through `sample` with a pair of variables
```
S1 = [S[key] for key, value in sample if value == 1]
```

You can test it out yourself with
```
for sample in sample_set:
    for key, value in sample:
        print("key: {}, value: {}".format(key, value))
```

Reply:
When I try your suggestion I get:

Traceback (most recent call last):
  File "npp.py", line 38, in <module>
    S1 = [S[key] for key, value in sample if value == 1]
  File "npp.py", line 38, in <listcomp>
    S1 = [S[key] for key, value in sample if value == 1]
TypeError: cannot unpack non-iterable int object

Reply:
Code:

for sample in sample_set:
    S1 = [S[key] for key, value in sample if value == 1]
    S0 = [S[key] for key, value in sample if value == 0]
    print("S0 Sum: ", sum(S0), "\tS1 Sum: ", sum(S1), "\t", S0)

Reply:
Modified to use first suggestion